nucypher/network/server.py

```diff
@@ -84,12 +85,24 @@ def make_rest_app(
         log=Logger("http-application-layer")
         ) -> Tuple:
 
-    forgetful_node_storage = ForgetfulNodeStorage(federated_only=this_node.federated_only)
+    # A trampoline function for the real REST app,
```

Comment:
This is meant as a docstring? Perhaps include `"""`

Reply:
Not really, it's not something a user of it should see. The only important part is that the returned `rest_app` doesn't hold a reference to `datastore`, but I would think that's the default. The docstring would be whatever the original `make_rest_app()` had, and it didn't have any. I can add one though.

Reply:
Added a docstring and some type annotations.